lightblue-ldap-integration-test/pom.xml

```diff
@@ -49,13 +45,13 @@
         <dependency>
             <groupId>com.redhat.lightblue.mongo</groupId>
             <artifactId>lightblue-mongo-config</artifactId>
-            <version>${lightblue.mongo.version}</version>
+            <version>1.3.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>com.redhat.lightblue.mongo</groupId>
             <artifactId>lightblue-mongo-test</artifactId>
-            <version>${lightblue.mongo.version}</version>
+            <version>1.3.0-SNAPSHOT</version>
```

Comment:
Here I pull in mongo for testing purposes. I assume the release plugin will get this also?